docs/content/latest/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md

```diff
@@ -49,10 +49,10 @@ To restore YugabyteDB universe YCQL data from a backup, follow these steps.
     - **Keyspace**: Specify the keyspace.
     - **Table**: Specify the table to be restored. Note: The table name must be different than the backed up table name.
     - **Parallel Threads**: Default is `8`. This value can be change to a value between `1` and `100`.
-    - **KMS Configuration**: (optional) If the backup was from a universe that was encrypted at rest, then select the KMS     configuration to use.
+    - **KMS Configuration**: (optional) If you are restoring data from a universe that has [encryption at rest enabled](../secure/enable-encryption-at-rest), then you must select the KMS configuration to use so that the universe keys referenced in the metadata file can be retrieved. The retrieved keys are used to build and augment the key registry on the restore universe with the required universe keys. The universe data files are restored as normal afterwards.
```

Comment:
Should be has/had encryption at rest enabled. Since enabling/disabling encryption at rest or rotating a universe key are asynchronous, even if the universe was previously encrypted at rest but is not anymore, we cannot guarantee that no data files are encrypted anymore (even though no new data files are being encrypted).